### PR TITLE
XW-4155 | Remove jquery usage from mercury_ads_js

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

File: resources/wikia/modules/tracker.stub.js [hunk 1]
```diff
@@ -237,7 +237,11 @@ function veTrack( data ) {
 			defaultData.whichVE = 'unknown';
 		}
 
-		finalData = $.extend( {}, defaultData, data );
+		if (window.$) {
```
Comment (rybmat) [inline resources/wikia/modules/tracker.stub.js]: a comment why we leave jquery in this place would be awesome

File: extensions/wikia/AdEngine/js/template/floor.js [hunk 2]
```diff
@@ -93,22 +90,24 @@ define('ext.wikia.adEngine.template.floor', [
...
 		}
 
-		if (!isFloorPresent) {
-			$floor = $(floorHtml);
-			$floor.addClass('hidden');
-			$(doc.body).append($floor);
+		if (!floor) {
+			floor = document.createElement('div');
+			floor.id = floorId;
+			floor.classNames = 'hidden';
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/template/floor.js]: classNameS?

**Review by kvas-damian — CHANGES_REQUESTED** [review]
Please fix the typo

**Review by kvas-damian — APPROVED** [review]

File: extensions/wikia/AdEngine/js/template/floor.js [hunk 3]
```diff
@@ -70,13 +65,15 @@ define('ext.wikia.adEngine.template.floor', [
...
-			$floor.removeClass('hidden');
+			floor.classList.remove('hidden');
 		}
 
 		if (params.onClick) {
-			$(iframe).on('load', function () {
-				var iframeDoc = iframe.contentWindow.document;
-				$('html', iframeDoc).css('cursor', 'pointer').on('click', params.onClick);
+			iframe.addEventListener('load', function () {
+				var iFrameBody = iframe.contentWindow.document.body;
```
Comment (latata) [inline extensions/wikia/AdEngine/js/template/floor.js:73]: did you test it? I'm always worried about iframe manipulations.

File: extensions/wikia/AdEngine/js/template/floor.js [hunk 4]
```diff
@@ -93,22 +90,24 @@ define('ext.wikia.adEngine.template.floor', [
...
 		}
 
-		$floor.find('a.close').click(function (event) {
+		floor.querySelector('a.close').addEventListener('click', function (event) {
 			event.preventDefault();
-			$floor.addClass('hidden');
+			floor.classList.add('hidden');
 		});
-		$floor.find('.ad').html(iframe);
+		floor.querySelector('.ad').appendChild(iframe);
```
Comment (latata) [inline extensions/wikia/AdEngine/js/template/floor.js:110]: are you sure `.ad` is empty before appneding?

File: extensions/wikia/PaidAssetDrop/js/paidAssetDrop.js [hunk 5]
```diff
@@ -134,12 +134,26 @@ define('ext.wikia.paidAssetDrop.paidAssetDrop', [
 
 		log('Sending request to: ' + url, 'debug', logGroup);
 
-		$.ajax({url: url, dataType: 'json'}).then(function (response) {
+		var padRequest;
+
+		// This file is loaded both in Oasis and mobile-wiki
+		// in mobile-wiki we do not have jquery but we do have a fetch polyfill so it is save to use this
```
Comment (latata) [inline extensions/wikia/PaidAssetDrop/js/paidAssetDrop.js:140]: do we have polyfill in `window` in MobileWiki?

**Pull request merged**